modules/cherry-helpers/helpers/helper-view.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Helper View class file
+ */
+
+/**
+ * Helper View class
+ */
+class Helper_View {
+
+	/**
+	 * Render
+	 *
+	 * @param type  $__path storage view path.
+	 * @param  array $__data include data.
+	 * @return rendered html
+	 */
+	public static function render( $__path, $__data ) {
+		ob_start();
+
+		// Extract view datas.
+		extract( $__data );
```

Comment:
[Wordpress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#dont-extract) don't recommend using the `extract` function:

> extract() is a terrible function that makes code harder to debug and harder to understand. We should discourage it’s use and remove all of our uses of it.